src/main/java/org/apache/pulsar/manager/utils/HttpUtil.java

```diff
@@ -130,7 +130,7 @@ public static void initHttpClient() {
         }
     }
 
-    public static String doGet(String url, Map<String, String> header){
```

Comment:
This is a breaking change. While this class is probably an internal util - it is still marked public and well, people will likely be using it even if they are not "supposed to" :) 

Unless it is really needed I would just go back to static access. My 2c.

Reply:
I removed static from most of these methods because I wanted to get rid of powermock which succeeded.
It's also not the best design to make things static while it can perfectly be a bean and depending classes it's test can rely on mockito to mock it out, like I did when I replaced it with powermock.

I don't see why this is a breaking change, who is using this class besides some internal classes within this project? This is not a library.

Reply:
> I removed static from most of these methods because I wanted to get rid of powermock which succeeded.
It's also not the best design to make things static while it can perfectly be a bean and depending classes it's tests can rely on mockito to mock it out, like I did when I replaced it with powermock.

I completely agree w/ all of the reasoning. 

> who is using this class besides some internal classes within this project? 

Agreed, it would only be a breaking change if it is being used elsewhere. I don't think its clearly labeled internal. I am ok w/ making that "label" now though w/ these changes.

Reply:
I am ok w/ assuming this is an internal only class.